spyder/widgets/mixins.py

```diff
@@ -517,7 +533,8 @@ def show_hint(self, text, inspect_word, at_point):
 
         self.show_tooltip(signature=html_signature, text=extra_text,
                           at_point=point, inspect_word=inspect_word,
-                          display_link=True, max_lines=10, cursor=cursor)
+                          display_link=True, max_lines=max_lines, width=width,

```

Comment:
Perhaps in the case of tooltips we should show more lines (30?) and make the wrapping wider (120?) to not lose the important information they give to the user.

But please test things with other LSPs to see how to adjust that.